django_genericfilters/forms.py

```diff
@@ -66,6 +68,13 @@ def get_order_by_choices(self):
             _("Don't forget to implements get_order_by_choices"))
 
 
+def clean_yesno(value):
+    if isinstance(value, string_types):
```

Comment:
Is it just a micro-optimisation ?

Reply:
No, just prevent an issue when the value is a list or tuple (or whatever).

Reply:
Ok, right, `get()` will fail if called with a list